lib/package.gi

```diff
@@ -182,21 +182,21 @@ BindGlobal( "FindPackageInfosInSubdirectories", function( pkgdir, name )
     fi;
 
     file:= Filename( [ pkgdir ],
-                      Concatenation( name, "/PackageInfo.g" ) );
+                      StringFormatted( "{}/PackageInfo.g", name ) );
```

Comment:
Personally, I find the old code both shorter and more readable.

Reply:
I would agree with @fingolfin. I would say that if is really just joining strings together then `Concatenation` is more natural, but if it also involves converting other data to a string, as in the not uncommon:
```
Concatenation("foo :",String(foo)," etc.\n");
```
then `StringFormatted` is better.

Reply:
I would use `StringFormatted` only if nontrivial formatting is involved.
For the situation mentioned by @stevelinton (concatenation after conversion of objects to strings if necessary), I would regard the currently undocumented `STRINGIFY` as more appropriate; perhaps `STRINGIFY` could become documented, with a nicer name.